src/generic/hooks.js

```diff
@@ -19,3 +21,25 @@ export function useEventListener(type, handler) {
     return () => global.removeEventListener(type, eventListenerRef.current);
   }, [type, handler]);
 }
+
+export function useIFramePluginEvents(onIframeLoaded) {
+  const [iframeHeight, setIframeHeight] = useState(0);
```

Comment:
```suggestion
  const [iframeHeight, setIframeHeight] = useState(null);
```

When the default of `0` is used, it results in the iframe having an attribute of `height=0`. Setting this to `null` prevents the height attribute being rendered in the DOM.

I am not sure of the technical benefits of this in the browser. But I was confused as while debugging for a second and thought maybe the discussions app is sending wrong height in the `postMessage`.

Reply:
Thanks! That sounds like a good change I've applied it.